cloudify_cli/commands/teardown.py

```diff
@@ -105,7 +105,7 @@ def _update_local_provider_context(ctx, manager_ip, logger):
 
 
 def _get_number_of_deployments(manager_ip):
-    client = env.get_rest_client(manager_ip)
```

Comment:
just `get_rest_client('blabla')` uses it as the profile object - first argument to `get_rest_client`